### ISSUE TITLE
Shared Git Credentials

### ISSUE THREAD

**Issue by grantjk** [czechboy0/Buildasaur#182]
Every new syncer requires the same github token and keys be set again and again. It would be nice if these could be set once and re-used as appropriate.

**Comment by czechboy0** [comment]
GitHub tokens and SSH keys are associated with a project - which are saved separately, so if you're creating more than one syncer with the same project, you should be able to reuse the whole project from the selection.

If you have _multiple projects_ on GitHub, but use the same SSH keys and GitHub token, unfortunately there's not much I can do. I'll try to see if I can make the setup a bit faster with the current constraints.

**Comment by grantjk** [comment]
Yup - we have mulitple projects and our build server has a single set of ssh keys with access to all of them.
